staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiserver.go

```diff
@@ -274,6 +324,9 @@ func (s *APIAggregator) AddAPIService(apiService *apiregistration.APIService) {
 	}
 	proxyHandler.updateAPIService(apiService)
 	s.proxyHandlers[apiService.Name] = proxyHandler
+
+	s.deferLoadAPISpec(apiService.Name)
```

Comment:
Ah ha.  This is what broke kops.  No reason for you to run before the proxy path handling completes.